### ISSUE TITLE
please add ios platform

### ISSUE THREAD

**Issue by mrthuan** [dreamsoftin/facebook_audience_network#33]
please add ios platform

**Comment by kellvembarbosa** [comment]
UP, please add ios

**Comment by WieFel** [comment]
+1

**Comment by SachinGanesh** [comment]
Hi all, iOS support is now available. Please check "ios" branch. If everything works fine we will release it through pub.dev

You can use the below dependecy to test the integration
```yaml
facebook_audience_network: 
    git: 
      url: git@github.com:dreamsoftin/facebook_audience_network.git
      ref: ios
```

**Comment by malikdoksoz** [comment]
Please make the announcement so that everyone can see it, so let's speed up find bug the ios. Thank you :)

> Hi all, iOS support is now available. Please check "ios" branch. If everything works fine we will release it through pub.dev
> 
> You can use the below dependecy to test the integration
> 
> ```yaml
> facebook_audience_network: 
>     git: 
>       url: git@github.com:dreamsoftin/facebook_audience_network.git
>       ref: ios
> ```

Please make the announcement so that everyone can see it, so let's speed up find bug the ios. Thank you 👍

**Comment by kamy22** [comment]
Hi @SachinGanesh, thanks for your work. Your plugin works great on Android. I am testing the iOS branch and the first time I run `flutter build` I received the error:

```
[!] CocoaPods could not find compatible versions for pod "FBAudienceNetwork":
      In Podfile:
        facebook_audience_network (from `.symlinks/plugins/facebook_audience_network/ios`) was resolved to 0.0.1, which depends on
          FBAudienceNetwork (= 5.8.0)

    Specs satisfying the `FBAudienceNetwork (= 5.8.0)` dependency were found, but they required a higher minimum deployment target.
```

I changed the target version of the project from `iOS 8` to `iOS 9` to make it work correctly (by default flutter creates a project compatible with iOS 8). I think it should be mentioned in the `README.md`.

**Comment by SachinGanesh** [comment]
@kamy22  Thank you for pointing out. Will add it to the readme file

**Comment by iamar1k92** [comment]
flutter: Error: {error: No fill, placement_id: $$$$$$, invalidated: false}

even i have added the test id on the backend account.

**Comment by SachinGanesh** [comment]
Try the test id's given in example projects 
example IMG_16_9_APP_INSTALL#2312433698835503_2964952163583650

**Comment by SachinGanesh** [comment]
iOS support is available from version 0.6.0 [pub.dev](https://pub.dev/packages/facebook_audience_network)
